CircleciScripts/IntegrationTestsConfiguration.json

```diff
@@ -110,6 +110,12 @@
       "AWSKinesisVideoTests": {}
     }
   },
+  "KinesisVideoSignaling": {
+    "test_list": {
+      "AWSKinesisVideoSignalingTests": {},
+      "AWSKinesisVideoSignalingUnitTests": {}
```

Comment:
looks like the run_unittest_bundle runs the unit test so this can probably be removed? it doesn't look like the other sections in this file are listing the unit test targets either

Reply:
Whoops, good catch, removed